README.md

```diff
@@ -226,6 +226,15 @@ index them run:
 make reindex
 ```
 
+### 2.6 Cancelling all ongoing (re)indexing operations
+
+
+1) Go to the Simple Queue Service dashboard in the AWS Console. Then, find your target notify SQS queue (should be named azul-notify-...). 
+2) Go to the Lambda dashboard in the AWS Console. Find the azul-indexer...-write lambda. Then, disable the event binding document queue (usually named azul-documents-...). This is done by clicking on the `SQS` trigger in the `Designer` box, clicking on the Enabled switch of azul-documents-... in the newly appeared `SQS` box, then finally saving your settings.
+3) Purge the remaining queues.
+4) Renable the event binding from step 2.
+5) Repeat until azul-documents-..., azul-documents-....fifo, azul-notify-... is empty.
```

Comment:
Repeat what? Everything? That's not necessary. Mention a concrete step from which to repeat.

Reply:
Shouldn't atleast steps 2-4 be repeated since there are times the `azul-documents-...` and `azul-documents-....fifo` are still not empty after step 4. Also, I should have prepended ` If azul-documents-... and azul-documents-....fifo, isn't empty,...` to Step 5.